README.md

```diff
@@ -33,4 +33,4 @@
 
 ## Team Members
  1. [Saumya Tiwari](https://github.com/Saumya29)
- 2. [Nikhil Arora](https://github.com/iamoperand)
\ No newline at end of file
+ 2. [Nikhil Arora](https://github.com/iamoperand)
```

Comment:
Also include the steps to setup the boilerplate here (in Readme).

Reply:
On it.